Tests/test_file_png.py

```diff
@@ -79,7 +79,7 @@ def get_chunks(self, filename):
 
     def test_sanity(self, tmp_path):
         # internal version number
-        assert re.search(r"\d+\.\d+\.\d+(\.\d+)?$", features.version_codec("zlib"))
+        assert re.search(r"\d+(\.\d+){1,3}$", features.version_codec("zlib"))
```

Comment:
Without this change, our test suite fails with zlib 1.3, as this code presumes zlib versions are X.Y.Z, not just X.Y.

The Cygwin builds have actually started failing on main because this is missing - https://github.com/python-pillow/Pillow/actions/runs/5921422961